website/static/css/custom.css

```diff
@@ -5,3 +5,47 @@
 table td:first-child > code {
   white-space: nowrap;
 }
+
+pre {
+  position: relative;
+}
+
+pre .btn-icon {
+  position: absolute;
+  top: -34px;
+  z-index: 2;
+  cursor: pointer;
+  border: 1px solid transparent;
+  padding: 0;
+  color: #b0a673;
+  background-color: transparent;
+  height: 34px;
+  transition: all .25s ease-out;
+}
+
+pre .btn-icon:hover {
+  color: #000;
+  text-decoration: none;
+}
+
+.btn-icon__body {
+  align-items: center;
+  display: flex;
+}
+
+.btn-icon svg {
+  fill: currentColor;
+  margin-right: .4em;
+}
+
+.btn-icon__label {
+  font-size: 11px;
+}
+
+.btn-open-in-repl {
```

Comment:
Not needed

Reply:
Fixed, sorry I shouldn't have left all of this in, just wanted to post what I had functional at EOD.